yarn-project/noir-protocol-circuits/src/crates/types/src/abis/complete_address.nr

```diff
@@ -1,26 +1,29 @@
-use crate::point::Point;
-use crate::address::AztecAddress;
-use crate::hash::{compute_partial_address,compute_contract_address_from_partial};
+use crate::{
+    address::{
+        AztecAddress,
+        PartialAddress,
+    },
+    grumpkin_point::GrumpkinPoint,
+};
 
-struct CompleteAddress{
+struct CompleteAddress {
     address : AztecAddress,
-    public_key : Point, 
-    // TODO(David): Can we type this as AztecAddress instead of Field?
-    partial_address: Field,
+    public_key : GrumpkinPoint, 
+    partial_address: PartialAddress,
 }
 
 impl CompleteAddress{
     fn assert_is_zero(self) {
         self.address.assert_is_zero();
         self.public_key.assert_is_zero();
-        assert(self.partial_address == 0);
+        self.partial_address.assert_is_zero();
     }
     
-    pub fn compute(point : Point, contract_address_salt : Field, function_tree_root : Field, constructor_hash : Field) -> CompleteAddress {
-        let partial_address = compute_partial_address(contract_address_salt, function_tree_root, constructor_hash);
+    pub fn compute(point : GrumpkinPoint, contract_address_salt : Field, function_tree_root : Field, constructor_hash : Field) -> CompleteAddress {
+        let partial_address = PartialAddress::compute(contract_address_salt, function_tree_root, constructor_hash);
         
         CompleteAddress{
-            address : compute_contract_address_from_partial(point, partial_address),
+            address : AztecAddress::compute(point, partial_address),
```

Comment:
Should the `point` here be a `pub_key` instead? To keep naming more consistent. In `compute` it is a public key.

Reply:
Agree, addressed in https://github.com/AztecProtocol/aztec-packages/pull/3814/commits/10dcf22cd3624bbec700cabe3ee5ac1ed7330f6e